LogMonitor/src/LogMonitor/LogWriter.h

```diff
@@ -21,7 +21,7 @@ class LogWriter final
 
         m_isConsole = true;
 
-        _setmode(_fileno(stdout), _O_U8TEXT);
+       // _setmode(_fileno(stdout), _O_U8TEXT);

```

Comment:
we need to figure out how to solve this, merging it as is would make us loose Unicode support

Reply:
Looking into this!

Reply:
@bobsira please convert this to a draft or create a task to track the utf8 bug before we can merge this